tv_apps/smart-home/js/home.js

```diff
@@ -640,26 +662,39 @@
       this.spatialNavigator.remove(this.folderScrollable);
       this.folderScrollable.clean();
       this._folderCard = undefined;
+      this.edit.isFolderReady = false;
```

Comment:
It's quite hack but I have no good idea for this :-/
Maybe it's time to inject `home.js` into `edit.js` and let it just access `Home._folderCard`, but we can left it later.
After all please add `XXX` on declaration of `this.edit.isFolderReady`.